schemas/categories.txt

```diff
@@ -0,0 +1,67 @@
+- Barrel Scanner
+A fixture which has a rotating cylinder of mirrors to reflect the light in a moving pattern.
+Example: american-dj/inno-pocket-fusion = https://www.adj.com/inno-pocket-fusion
+
+- Blinder
+Extremely bright fixtures intended to light the audience, rather than the stage.
+Example: elation/cuepix-blinder-ww2 = https://www.elationlighting.com/cuepix-blinder-ww2
+
+- Color Changer
+A fixture which can change its color electronically, especially (but not only) one with a gel wheel.  This includes RGB LEDs.
+Example: clay-paky/sharpy = https://www.claypaky.it/en/products/sharpy
+
+- Dimmer
+A device which takes mains power and DMX, and outputs PWM dimmed power, like for a conventional fixture to use.
+Example: chauvet-dj/dmx-4 = https://www.chauvetdj.com/products/dmx-4/
```

Comment:
Yes, dimmer packs are in that category currently. But most "Dimmer" fixtures are "simple" PAR lights that are mainly used for (non-colored) illumination of e.g. a stage.

It might make sense to split this category into two.

Reply:
FWIW, my rationale for making "Dimmer" mean "dimmer pack" and not "any fixture with an intensity feature" was:
 1. nearly all fixtures have dimming capability, and
 2. you can already tell if a fixture can dim via DMX by searching for an "Intensity" channel

If we keep both dimmer packs and dimmable fixtures in the same category, it's not a terribly useful category.  If we kick out dimmable fixtures, and they don't have any other specific features, then they have to be "Other" (as each fixture is required to have at least one category).  So I'm tentatively inclined to agree that they need their own category.

(Currently, fewer than 10% of fixtures in OFL *don't* have an Intensity channel -- mostly smoke/hazers, plus some lasers and pyro and other oddballs.)

I'm open as to what this new category should be called, or how it should be defined.  Do we want a generic "controllable intensity" that everybody should use?  Or a "dimmable white light" for these PARs which would otherwise not have any specific category?

Reply:
My 2c, on reading this today: if we had categories for how fixtures are described and used in practice (like PAR, Beam, Spot, Wash, Cyc), then we wouldn't need a new category for "has an intensity channel".

From a programming perspective, if you wanted all your washes, you'd tell your console to grab everything where "Category contains Wash".  If you wanted everything with controllable intensity, you'd ask for "every fixture where Channel contains type=Intensity".  There's no case I can think of where I'd want "Category contains Dimmable".  That's just another redundant tag which is going to occasionally get missed.